systemd/nslogin/Makefile

```diff
@@ -0,0 +1,18 @@
+SRC=nslogin.c
+LIBS=-lsystemd
+CFLAGS=-Wall -Wextra -Wconversion -pedantic
+CMODEFLAGS=-O0 -g -DDEBUG
+POSTBUILD=sudo chown root:root $@ && sudo chmod a+s $@
+LINTER=clang-tidy-12
+
```

Comment:
Can you add an install target here and a make at the root of the project too, installing compiled and non compiled files relative to `DESTDIR` and `PREFIX`?

Reply:
Please check if I got it as you thought.

Reply:
I did try a package build (`debuild` or `dpkg-buildpackage`) and it seems that DESTDIR is overridden (it tries to install those in `/usr/libexec`). If you don’t have the time to look at it. I can give it a look (probably Wednesday), so don’t kill yourself on this :)